packages/electrode-archetype-react-app/README.md

```diff
@@ -236,6 +236,35 @@ var supports = require("electrode-archetype-react-app/supports");
 
 supports.cssModuleHook();
 ```
+## Extending Webpack Configuration
+
+The webpack config for your app is being generated by the archetype. While composing the webpack config, the archetype looks for the `archetype/config` folder at the project root of your app.
+In order to override or extend the webpack configuration you should create a file under `archetype/config/webpack`.
```

Comment:
Is this path `archetype/config/webpack` actually correct?

In https://github.com/electrode-io/electrode/issues/97#issuecomment-274150029 @ananavati mentioned that webpack configuration files in the root of the app will be used to extend the configuration. I've tested this to be working 2-3 weeks ago.

Has this been changed?